setup.cfg

```diff
@@ -48,6 +48,7 @@ install_requires =
     django-rest-framework>=0.1.0
     djangorestframework>=3.14.0
     django-environ>=0.10.0
+    django-ninja==1.2.1
```

Comment:
add pydantic and on put a minimum version for this.

Reply:
Done!